components/price-stabilization/DebtCoupon.tsx

```diff
@@ -28,31 +29,32 @@ type Coupons = {
 const uDEBT = "uDEBT";
 const uAR = "uAR";
```

Comment:
@Zequez is this visible? https://github.com/ubiquity/ubiquity-dollar-development/issues/146

Reply:
Line 483 it looks like uDEBT is still rendered in the UI lol